README.md

```diff
@@ -92,18 +92,22 @@ $ DOCKERFILES_ONLY=1 ./bin_build_containers.sh
 
 ## Running the Unit Tests
 
-Start the containers.
+Build and start the containers.
 
 ```bash
-$ ./bin/build_containers.sh
+$ SKIP_SERVER=1 \
+  SKIP_WEBSOCKET=1 \
+  SKIP_NGINX=1 \
+  SKIP_VIEWER=1 \
+  SKIP_WIFI_CONNECT=1 \
+  ./bin/build_containers.sh
 $ docker compose \
     -f docker-compose.test.yml up -d
 ```
 
 Run the unit tests.
 
 ```bash
-$ ./bin/build_containers.sh
```

Comment:
@vpetersson, removed this one as `./bin/build_containers.sh` is already specified in Line 103 (see the right panel for details).